lutris/runners/winesteam.py

```diff
@@ -128,7 +128,7 @@ def appid(self):
 
     @property
     def prefix_path(self):
-        _prefix = self.game_config.get('prefix') or self.get_default_prefix()
+        _prefix = self.game_config.get('prefix') or self.get_or_create_default_prefix(arch=self.game_config.get('arch'))
```

Comment:
Does Lutris (aspire to) adhere to the PEP 8 line length limit? If so, this line needs to be split.